src/app/scheduler/components/SchedulerCalendar.tsx

```diff
@@ -109,6 +110,15 @@ export function SchedulerCalendar({ calendarEvents }: SchedulerCalendarProps): J
     return today;
   }, [term]);
 
+  const getTermMonthLowerBound = useCallback(() => {
+    const month = /\d{4}(\d{2})/.exec(term);
+    const year = /(\d{4})\d{2}/.exec(term);
+
+    const lowerBound = dayjs.utc(`${month ? month[1] : '09'}, 1, ${year ? year[1] : 2021}`, 'MM, D, YYYY');
```

Comment:
instead of defaulting to `09` lets default to `01`. I know it's pretty much impossible for it to happen, with `09` as the default, if it's the summer or winter term, the lower bound will be after the term ends.

Reply:
can't add a comment there but on line 258 you're missing `getTermMonthLowerBound` from the dependency array. If adding it breaks the program, you can add an eslint ignore so the linter doesn't get mad